libraries/AP_Baro/AP_Baro_config.h

```diff
@@ -103,3 +103,7 @@
 // this allows for using the simple model with the --ekf-single configure option
 #define AP_BARO_1976_STANDARD_ATMOSPHERE_ENABLED HAL_WITH_EKF_DOUBLE || AP_SIM_ENABLED
 #endif
+
+#ifndef HAL_BARO_THST_COMP_ENABLED
+#define HAL_BARO_THST_COMP_ENABLED 1
```

Comment:
Need to know how useful this might be on other vehicles.  And flash cost...

Reply:
Cost 160 bytes avg on Durandal

Reply:
Ok fixed